contrib/buildsystems/CMakeLists.txt

```diff
@@ -178,7 +192,9 @@ endif()
 
 find_program(MSGFMT_EXE msgfmt)
 if(NOT MSGFMT_EXE)
-	set(MSGFMT_EXE ${CMAKE_SOURCE_DIR}/compat/vcbuild/vcpkg/downloads/tools/msys2/msys64/usr/bin/msgfmt.exe)
+	if (USING_VCPKG)
+		set(MSGFMT_EXE ${CMAKE_SOURCE_DIR}/compat/vcbuild/vcpkg/downloads/tools/msys2/msys64/usr/bin/msgfmt.exe)
+	endif()
```

Comment:
While you are already in this area, something I noticed a while ago is that we're trying to set `MSGFMT_EXE` even if `NO_GETTEXT` was configured, which does not make sense.

Maybe I can talk you into ensuring that `MSGFMT_EXE` is unset in that case, e.g. by surrounding this entire block with:

```cmake
if(NO_GETTEXT)
    if(MSGFMT_EXE)
        message(WARNING "MSGFMT_EXE ignored under NO_GETTEXT")
        unset(MSGFMT_EXE)
    endif()
else()
    ... <current block> ...
endif()
```

Reply:
Yeah I can do that

Reply:
I've added in the code as you've asked, I'm not super familiar with how its supposed to act with the Intl library, since if that gets found it sets NO_GETTEXT to 1 and it forces NO_GETTEXT to 1 later in the code.  So I'm not sure if it's ever actually possible to have Intl=1 and NO_GETTEXT=0 (or unset or whatever) at the same time.